projects/angular/src/data/datagrid/datagrid-row.spec.ts

```diff
@@ -134,6 +134,20 @@ export default function (): void {
         TestBed.get(Items).all = [{ id: 1 }, { id: 2 }];
       });
 
+      it('renders correctly if item is set after clrDgSelectable', fakeAsync(function () {
+        selectionProvider.selectionType = SelectionType.Multi;
+        const tempItem = context.testComponent.item;
+        context.testComponent.item = undefined;
+        context.detectChanges();
+        context.testComponent.clrDgSelectable = false;
```

Comment:
In some cases, the clrDgSelectable Input is called before clrDgItem  value. This test replicates this by forcing the item to be undefined, and then setting the clrDgSelectable, and then repopulating the item.